extension/src/fileSystem/index.ts

```diff
@@ -276,8 +277,9 @@ export const addPlotToDvcYamlFile = (cwd: string, plotObj: PlotConfigData) => {
   if (!plots?.range) {
     const plotYaml = getPlotsYaml(plotObj, dvcYamlLines)
     dvcYamlLines.push(...plotYaml)
-    writeFileSync(dvcYamlFile, dvcYamlLines.join('\n'))
-    return
+
+    void openFileInEditor(dvcYamlFile)
+    return writeFileSync(dvcYamlFile, dvcYamlLines.join('\n'))
```

Comment:
There is an empty line on top of the dvc.yaml in the case of there being an empty file:

```

plots:
  - step vs acc:
      template: confusion
      x: step
      y:
        training/plots/metrics/train/acc.json: acc

```

If we wanted to get rid of the top line, we could need to add an extra `if` check. I decided to just keep it to avoid extra complexity.